lib/tropic/tscroll.flow

```diff
@@ -1294,6 +1296,7 @@ createTScrollBars(
 					)
 				)
 				|> (\t -> TFixSize(t, makeScrollbarBox(boxSize)))
+				|> (\t -> TBorderTDir(zero, zero, if (horizontal) xScrollGap else zero, zero, t, isRTL))
```

Comment:
It would be better to move the vertical scrollbar to the left here instead of moving the whole group in #621 and compensate for horizontal here.